en/identity-server/next/docs/guides/multitenancy/manage-tenants.md

```diff
@@ -1,27 +1,41 @@
-# Tenant management
+# Manage Root Organizations (Tenants)
 
-By default,  {{ product_name }} is configured with a single tenant named `carbon.super`. If you have the required permissions, you can configure more tenant domains and maintain multiple isolated user bases on a single server/cluster at the same time.
+!!! Note
+    From now on, **Tenants** will be referred to as **Root Organizations**.
+
+    This change is to ensure consistency across our documentation and to better reflect the hierarchical structure of our organization system.
+
+By default,  {{ product_name }} is configured with a single root organization named `carbon.super`. If you have the required permissions, you can configure more root organizations and maintain multiple isolated user bases on a single server/cluster at the same time.
 
 !!! Info
     Learn more about [multi-tenancy]({{base_path}}/guides/multitenancy/).
 
-## Add new tenants
+## Add new root organizations
 
-Follow the steps given below to add a new tenant from the Management Console.
+Follow the steps given below to add a new root organization from the Console.
 
 !!! Note
-    You can access the Management Console using the following URL: `https://<hostname>:<port>/carbon`.
+    You can access the Console using the following URL: `https://<IS_HOST>:<PORT>/console`.
 
-1.  Sign in to the super tenant on the Management Console.
+1.  Sign in to the super tenant on the Console.
 
     !!! info 
-        To add and manage tenants using the Management Console, you need to be signed in as a super user.
+        To add and manage tenants using the Console, you need to be signed in as a super user.
```

Comment:
```suggestion
        To add and manage tenants using the {{ product_name }} Console, you need to be signed in as a super user.
```

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4820/commits/129149a8ed3f189a6d69a63b37780762503ebf0a